bot.rb

```diff
@@ -24,7 +24,7 @@ def self.mineSweeperBot(game)
         c.server = "irc.twitch.tv"
         c.channels = ["#quantumproductions"]
         c.nick = "quantumproductions"
-        c.password = "oauth:micj4w45jt0haehn3i8n68v2s07nsi" 
```

Comment:
we should put these values in `ENV` anyway.  Take a look at [dotenv](https://github.com/bkeepers/dotenv) and just put these values in a `.env` file and add that to `.gitignore`.